frontend/src/pages/UpdateUser.tsx

```diff
@@ -16,13 +16,14 @@ import TopAppBar from '../components/TopAppBar';
 import { useState } from 'react';
 import { Faculty, UserDetails } from '../types';
 import { updateUser } from '../api';
-import { useNavigate } from 'react-router-dom';
+import { useNavigate, useLocation } from 'react-router-dom';
 
 export const UpdateUser = () => {
+  const { state } = useLocation();
+  const { name } = state;
   const [userDetails, setUserDetails] = useState<UserDetails>({
-    emojis: '',
+    name: name,
     faculty: Faculty.Business,
-    faveEat: '',
     interests: '',
```

Comment:
As mentioned above, let's set the ground work for this to become the edit user page also. Let's set this default value as the user.interests. If user.interests doesn't exist yet, set it to an empty string.